### PR TITLE
EOS-22091: Fix for MessageBus log file to allow log entries by any user

### PR DESCRIPTION
**Pull request by rahul-tripathi-git**
Signed-off-by: Rahul Tripathi <rahul.tripathi@seagate.com>

# Problem Statement
- Problem statement 

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/utils/message_bus/message_bus.py [hunk 1]
```diff
@@ -33,16 +33,15 @@ class MessageBus(metaclass=Singleton):
 
     def __init__(self):
         """ Initialize a MessageBus and load its configurations """
-        Conf.load('config_file', 'json:///etc/cortx/cortx.conf',
+        Conf.load('config_file', 'json:///etc/cortx/cortx.conf', \
```
Comment (rohit-k-dwivedi) [inline py-utils/src/utils/message_bus/message_bus.py:36]: Unnecessary \ can be avoided

**Review by rohit-k-dwivedi — APPROVED** [review]

File: py-utils/src/utils/message_bus/message_bus.py [hunk 2]
```diff
@@ -33,16 +33,15 @@ class MessageBus(metaclass=Singleton):
...
-        os.chmod("/var/log/cortx/utils/message_bus", stat.S_IRWXU | \
-                 stat.S_IRWXG | stat.S_IROTH | stat.S_IXOTH)
-        os.chmod("/var/log/cortx/utils/message_bus/message_bus.log", \
-            stat.S_IREAD | stat.S_IWRITE | stat.S_IRGRP | stat.S_IWGRP | \
-            stat.S_IROTH | stat.S_IWOTH)
+        if not Log.logger:
+            log_level = Conf.get('config_file', 'utils>log_level', 'INFO')
+            Log.init('message_bus', '/var/log/cortx/utils/message_bus', \
+                level=log_level, backup_count=5, file_size_in_mb=5)
+            os.chmod("/var/log/cortx/utils/message_bus/message_bus.log", \
```
Comment (rohit-k-dwivedi) [inline py-utils/src/utils/message_bus/message_bus.py:42]: please check for inconsistent use of single and double quotes

File: py-utils/src/utils/message_bus/message_bus.py [hunk 3]
```diff
@@ -35,14 +35,10 @@ def __init__(self):
...
             skip_reload=True)
-        log_level = Conf.get('config_file', 'utils>log_level', 'INFO')
-        Log.init('message_bus', '/var/log/cortx/utils/message_bus',
-            level=log_level, backup_count=5, file_size_in_mb=5)
-        os.chmod("/var/log/cortx/utils/message_bus", stat.S_IRWXU | \
-                 stat.S_IRWXG | stat.S_IROTH | stat.S_IXOTH)
-        os.chmod("/var/log/cortx/utils/message_bus/message_bus.log", \
-            stat.S_IREAD | stat.S_IWRITE | stat.S_IRGRP | stat.S_IWGRP | \
-            stat.S_IROTH | stat.S_IWOTH)
+        if not Log.logger:
```
Comment (sachinpunadikar) [inline py-utils/src/utils/message_bus/message_bus.py:38]: Add comment on why we are checking this & what will happen if logger exists

Comment (rahul-tripathi-git) [reply py-utils/src/utils/message_bus/message_bus.py:42]: Done.

File: py-utils/src/utils/message_bus/message_bus.py (hunk 2 above)
Comment (rahul-tripathi-git) [reply py-utils/src/utils/message_bus/message_bus.py]: Done

**Pull request merged**